Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.6.8'
+ruby '~> 2.6'
```

Comment:
have you run bundle install after changing this?

Reply:
Yep. At least on my machine, it didn't change the lock file.